daemon/src/order/maker.rs

```diff
@@ -134,13 +134,13 @@ impl Actor {
             }
         };
 
-        let (order_id, offer, quantity, leverage) = match order {
+        let (order_id, offer_id, quantity, leverage) = match order {
```

Comment:
🤔 Sorry, I cannot follow why this file was changes in this way.

I had a look at https://github.com/itchysats/itchysats/commit/e4f910f35070c47672c8e37e224b51e3b532e833#diff-4826e36fe738978b5d804d1cd212d9060a660e636c4c4a9552621a23ce260843 and there are changes to `daemon/src/order/maker.rs` but not the changes you are "reverting" here.